src/commands/apphosting-backends-list.ts

```diff
@@ -29,18 +27,30 @@ export const command = new Command("apphosting:backends:list")
     }
 
     const backends = backendRes.backends ?? [];
-    for (const backend of backends) {
-      const [backendLocation, , backendId] = backend.name.split("/").slice(3, 6);
-      table.push([
-        backendId,
-        backend.codebase?.repository?.split("/").pop() ?? "",
-        backendLocation,
-        backend.uri.startsWith("https:") ? backend.uri : "https://" + backend.uri,
-        datetimeString(new Date(backend.createTime)),
-        datetimeString(new Date(backend.updateTime)),
-      ]);
-    }
-    logger.info(table.toString());
+    printBackendsTable(backends);
 
     return backends;
   });
+
+/**
+ * Prints a table given a list of backends
+ */
+export function printBackendsTable(backends: apphosting.Backend[]): void {
+  const table = new Table({
+    head: TABLE_HEAD,
+    style: { head: ["green"] },
+  });
+
+  for (const backend of backends) {
+    const [backendLocation, , backendId] = backend.name.split("/").slice(3, 6);
+    table.push([
+      backendId,
+      backend.codebase?.repository?.split("/").pop() ?? "",
```

Comment:
can you include a sample repository format string as a line comment here

Reply:
added sample